docs/v0.5.x/writing-a-plugin.md

```diff
@@ -358,3 +358,10 @@ Then once the pipeline hook has run, the deployment context will look like this:
 ```
 
 And every pipeline hook run thereafter will be able to access the `uploadedAt` property.
+
+If you find yourself requiring properties from another plugin in the same hook as it was set, you can specify plugin ordering (and thus property availability) in package.json,
+```javascript
+"ember-addon": {
+   after: "ember-cli-deploy-funky-plugin"
+}
```

Comment:
I thought the same regarding the title, but you can also argue that it's actually part of the discussion on the context, 
maybe subheading instead?

regarding natural order: agreed